### DISCUSSION TITLE
Compatibility with Yi Home 1080p 24US

### DISCUSSION THREAD

**Discussion by ziat007** [alienatedsec/yi-hack-v5#351] [Announcements]
**Is your feature request related to a problem? Please describe.**
Compatibility with Yi Home 1080p

**Describe the solution you'd like**
 - Firmware Version: [e.g. 2.1.1_20171024151200 EU]
 - Device: [e.g., YI HOME 1080P]
 - Model: [e.g., 24US]
 
Original Yi hacks v2 was compatible with this.
https://github.com/bompo/yi-hack-v2

**Comment by alienatedsec** [comment]
I am not sure which camera version it is but it looks different to what I know - you could try loading the `yi_home_1080p` files, but if it doesn't flash then it is another chipset
